Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/Scripts/MSBuildUnityProjectExporter.cs

```diff
@@ -75,6 +78,38 @@ public static void ExportCommonPropsFile(IUnityProjectExporter exporter, Compila
             propsExporter.Write();
         }
 
+        public static void ExportTopLevelDependenciesProject(IUnityProjectExporter exporter, MSBuildToolsConfig config, DirectoryInfo generatedProjectFolder, UnityProjectInfo unityProjectInfo)
+        {
```

Comment:
nit: add summary

Reply:
Leaving for now, as this class is temporary so I can split off the logic and then adapt it in the appropriate place.

---
In reply to: [369676752](https://github.com/microsoft/MSBuildForUnity/pull/111#discussion_r369676752) [](ancestors = 369676752)